monai/data/meta_tensor.py

```diff
@@ -505,7 +505,7 @@ def peek_pending_rank(self):
         a = self.pending_operations[-1].get(LazyAttr.AFFINE, None) if self.pending_operations else self.affine
         return 1 if a is None else int(max(1, len(a) - 1))
 
-    def new_empty(self, size, dtype=None, device=None, requires_grad=False):
+    def new_empty(self, size, dtype=None, device=None, requires_grad=False):  # type: ignore[override]
```

Comment:
Does the `@override` decorator from `typing` not work here when the method signature isn't changed?

Reply:
Yes, it's not work.